### PR TITLE
Move Check for deprecated messages without sphinx directive to pre-commit

### PR DESCRIPTION
**Pull request by nooraLeila**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/core/arrays/categorical.py [hunk 1]
```diff
@@ -906,7 +906,7 @@ def set_categories(
            Whether or not to reorder the categories in-place or return a copy
            of this categorical with reordered categories.
 
-           .. deprecated:: 1.3.0
+           .. Deprecated:: 1.3.0
```
Comment (MarcoGorelli) [inline pandas/core/arrays/categorical.py:909]: we can revert this now we've checked it :)

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
nice!

File: pandas/core/arrays/categorical.py (hunk 1 above)
Comment (nooraLeila) [reply pandas/core/arrays/categorical.py:909]: yes I checked it on my side- not sure why it appeared here as I corrected it :woman_shrugging:

Comment (MarcoGorelli) [reply pandas/core/arrays/categorical.py:909]: also you can now remove this part of the check from `ci/code_checks.sh`

Comment (MarcoGorelli) [reply pandas/core/arrays/categorical.py:909]: > not sure why it appeared here as I corrected it

did you do `git add -a` and `git commit -m '<some message>'` after you'd corrected it?

**Comment by MarcoGorelli** [comment]
Hi Noora - are you still working on this? Need help with anything? (no hurry of course)

**Comment by nooraLeila** [comment]
HI Marco

Just spotted this! I’ll pick up the sprints in September. Go ahead and work on anything I began in the meantime.

-N

**Comment by MarcoGorelli** [comment]
thanks Noora, no worries - closing for now then

**Pull request closed without merging**